src/main/scala/karazin/scala/users/group/week4/homework/Homework.scala

```diff
@@ -35,23 +36,27 @@ object Homework:
 
     infix def contains(x: Int): Boolean = false
 
-    infix def remove(x: Int): IntSet = ???
+    infix def remove(x: Int): IntSet = this
     
     @targetName("union")
-    infix def ∪(that: IntSet): IntSet = ???
+    infix def ∪(that: IntSet): IntSet = that
 
     @targetName("intersection")
-    infix def ∩(that: IntSet): IntSet = ???
+    infix def ∩(that: IntSet): IntSet = Empty
 
     @targetName("complement")
-    infix def ∖(that: IntSet): IntSet = ???
+    infix def \(that: IntSet): IntSet = Empty
 
     @targetName("disjunctive union")
-    infix def ∆(that: IntSet): IntSet = ???
+    infix def ∆(that: IntSet): IntSet = that
     
     override def toString: String = "[*]"    
     
-    override def equals(other: Any): Boolean = ???    
+    override def equals(other: Any): Boolean =
```

Comment:
Broken contract on `hashCode`.

Reply:
Dear professor, I had add hashCode function. Please help me check it. Thanks.